### ISSUE TITLE
[Bug]: ERROR: could not extend file "base/" with FileFallocate()

### ISSUE THREAD

**Issue by fatihsazan** [timescale/timescaledb#6501]
### What type of bug is this?

Unexpected error

### What subsystems and features are affected?

Restore

### What happened?

While transfering data with the timescaledb-parallel-copy or normal copy command or using ora2pg with timescaledb extension, in particular hyper-tables,   we constantly receive the following error even though there is more space in the system.
We are using Postgres 16 and timescaledb 2.13.0.

Is there anyone who has this error, encountered this error, or has a suggestion for a solution?

 org.postgresql.util.PSQLException: ERROR: could not extend file "base/488318210/534150832" with FileFallocate(): No space left on device
   Hint: Check free disk space.
  Where: COPY ****, line 3044

### TimescaleDB version affected

2.13.0

### PostgreSQL version used

16

### What operating system did you use?

ubuntu 22.04.03

### What installation method did you use?

RPM

### What platform did you run on?

On prem/Self-hosted

### How can we reproduce the bug?

```bash
You can create any hypertable on  Postgresql 16 with timescaledb 2.3.0 . U can try import it u coulnd't get error first time if you import table more than one time probably u can reproduce the situation.
```

**Comment by antekresic** [comment]
Looks like you ran out of disk space necessary to copy in the data.

Can you double check if its that?

**Comment by fatihsazan** [comment]
Yes, I checked, we just started data transfer on a server we just set up.
65T   44T   22T  68% /pg-data   ->  %32 free space .

We were encountering this even when there was no data on the server. This error occurs when inserting data into a hypertable table, either at the beginning or in between, and the copy process stops. We encounter bulk copy when transferring data. For example, when assigning data file (csv format) to 50m - 100m rows in a day. When assigning to a hypertable table with a 3-hour chunk.
